KomaMRIFiles/test/runtests.jl

```diff
@@ -40,14 +40,64 @@ using TestItems, TestItemRunner
         obj = read_phantom_jemris(path*"/test_files/column1d.h5")
         @test obj.name == "column1d.h5"
     end
-    # Test JEMRIS
+    # Test MRiLab
     @testset "MRiLab" begin
         path = @__DIR__
         filename = path * "/test_files/brain_mrilab.mat"
         FRange_filename = path * "/test_files/FRange.mat" #Slab within slice thickness
         obj = read_phantom_MRiLab(filename; FRange_filename)
         @test obj.name == "brain_mrilab.mat"
     end
+    # Test Phantom (.phantom)
+    @testset "WritePhantom" begin
+        path = @__DIR__
+        # NoMotion
+        filename = path * "/test_files/brain_nomotion.phantom"
+        obj = KomaMRIFiles.brain_phantom3D()
+        obj.name = "brain2D_axial_nomotion"
+        write_phantom(obj, filename)
+        # SimpleMotion
+        filename = path * "/test_files/brain_simplemotion.phantom"
+        obj = KomaMRIFiles.brain_phantom3D()
+        obj.name = "brain2D_axial_simplemotion"
+        obj.motion = KomaMRIFiles.SimpleMotion([
+            KomaMRIFiles.PeriodicRotation(
+                period=1.0, 
+                yaw=45.0),
+            KomaMRIFiles.Translation(
+                ti=0.0,
+                tf=0.5,
+                dy=0.02
+        )])
+        write_phantom(obj, filename)
```

Comment:
If you want to test the `write_phantom` do `obj2 = read(write(obj1))` and check that `obj1 == obj2`.